multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/BuildCloudDeployModelStep.java

```diff
@@ -85,7 +90,7 @@ protected StepPhase executeStep(ProcessContext context) {
                                                                                     mtaModulesForDeployment);
         List<String> moduleJsons = modulesCalculatedForDeployment.stream()
                                                                  .map(SecureSerialization::toJson)
-                                                                 .collect(Collectors.toList());
+                                                                 .collect(toList());
```

Comment:
This is the first time in our code I've seen this. Why not use the standard way with `Collectors.<method>`?

Reply:
it's the same method, just put it as a static import